SAP/techne
# 857: Docs deployment to ghpages

This PR covers parts of #857 and #865.

Most of the CSS/JS for the docs were already working and this PR just updates the gulp tasks to copy over the techne resources in the right location. 

To test this - 

- run gulp docs-build from the terminal and verify if techne.css techne.svg are copied in ./docs/css
- run jekyll build --source ./docs and verify the _site folder is generated with techne.css and techne.svg and all other necessary assets to run the site.
